### PR TITLE
Added Fake Resume Generator and @jimrushing 's reCaptcha Solver

### PR DESCRIPTION
**Pull request by KathrynElrod**
Figured it would be pretty easy to filter out fake requests by looking at the old resume.png, so I added a resume generator. Adds fpdf, pdf2image, and poppler as dependencies.

### CODE REVIEW

**Comment by kylerchin** [comment]
Thank you very much!

@SeanDaBlack can you review and possibly merge?

**Review by es50678 — APPROVED** [review]
this is awesome. ran it a few times, seems to work.

**Comment by Audibles** [comment]
this is great!!

**Review by Audibles — APPROVED** [review]

**Comment by joeyagreco** [comment]
Could you generate a random resume file name and stash it to a variable in the code to fix [this issue](https://github.com/SeanDaBlack/KelloggBot/issues/7)?

File: req.py [hunk 1]
```diff
@@ -98,12 +102,17 @@ def generate_account(driver, rand_num):
...
+    print(f"successfully made account for fake email {fake_identity['email']}")
 
-def fill_out_application_and_submit(driver, rand_num):
+def fill_out_application_and_submit(driver, rand_num, fake_identity={'first_name': fake.first_name(), 'last_name': fake.last_name(), 'email': fake.free_email()}):
 
     driver.implicitly_wait(10)
     city = list(cities.keys())[rand_num]
+
+    # make resume
+    make_resume(fake_identity['first_name']+' '+fake_identity['last_name'], fake_identity['email'], 'resume.pdf')
```
Comment (joeyagreco) [inline req.py:113]: Referencing t[his comment](https://github.com/SeanDaBlack/KelloggBot/pull/17#issuecomment-991388174). For the 3rd parameter here, you could pass in a randomly generated resume file name

**Comment by KathrynElrod** [comment]
@joeyagreco How’s that?

File: req.py (hunk 1 above)
Comment (KathrynElrod) [reply req.py]: Added!

**Comment by KathrynElrod** [comment]
Resolved merge conflict - now using @MeaningOf42 's email generator

**Pull request merged**